docs/tutorial/Testing.md

```diff
@@ -34,6 +34,49 @@ test('reducer works as expected', (t) => {
 > not equal within JavaScript, and so are best to avoid if you want to compare
 > effects in your tests.
 
+## Working with nested cmd objects
+
+You may need to test objects from the `Cmd` module that make use of
+`Cmd.list` or `Cmd.map`. Imagine in the above example the reducer is updated to
+perform multiple commands when loading starts. We want to test that the properly
+shaped `fetchDetails` cmd is called, but don't necessarily want to mock out every
```

Comment:
what do you mean mock out effects? You shouldn't need to mock any effect because they are not run

Reply:
updated this language to be a bit more clear :+1: